projects/rulesets-base/src/extended-rules/spectral-rule.ts

```diff
@@ -186,8 +205,21 @@ export class SpectralRule extends ExternalRuleBase {
         );
       } else {
         // find if there is an appropriate change
-        const maybeChange: ObjectDiff | undefined =
+        let maybeChange: ObjectDiff | undefined =
           changesByJsonPath[fact.location.jsonPath];
+
+        if (
+          !maybeChange &&
+          jsonPointerHelpers.matches(fact.location.jsonPath, [
+            'paths',
+            '**',
+            `{${Object.values(OpenAPIV3.HttpMethods).join(',')}}`,
+          ])
+        ) {
```

Comment:
@acunniffe just an FYI, this is kind of weird but it's the same thing as the groupDiffs w/ path.

There's a few solutions to fixing this, one of which is some sort of internal spec format, the other being a better abstraction around the fact trees / jsonPath -> Fact type